### PR TITLE
AICORE-457: expose nuxeo.ai.insight.client.enable property and use it…

### PR DESCRIPTION
**Pull request by andreacornaglia**
… to render ai bulk btns

### CODE REVIEW

**Review by andreinechaev — APPROVED** [review]

File: nuxeo-ai-core-package/src/main/resources/install/templates/nuxeo-ai-core/config/ai-config-webui.xml.nxftl [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.web.ui.ai.properties.contrib">
+  <extension target="org.nuxeo.runtime.ConfigurationService" point="configuration">
+      <!-- AI Specific vars -->
+    <property name="org.nuxeo.web.ui.insight.client.enabled">
+      ${nuxeo.ai.insight.client.enable:=false}
+    </property>
+  </extension>
+</component>
```
Comment (nelsonsilva) [inline nuxeo-ai-core-package/src/main/resources/install/templates/nuxeo-ai-core/config/ai-config-webui.xml.nxftl:9]: nit: missing empty line

File: nuxeo-ai-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-ai/nuxeo-ai-bulk-add-button.html [hunk 2]
```diff
@@ -53,6 +55,13 @@
               status: {
                   type: Object,
                   notify: true,
+              },
+
+              isAiEnabled: {
```
Comment (nelsonsilva) [inline nuxeo-ai-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-ai/nuxeo-ai-bulk-add-button.html:60]: nit: we've been using a method `_isAvailable()` in our actions, perhaps it'd be nice to follow the convention

Comment (tiagocardoso) [inline nuxeo-ai-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-ai/nuxeo-ai-bulk-add-button.html:60]: I would call it "isInsightEnabled" as you could have 3rd party AI providers working.

File: nuxeo-ai-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-ai/nuxeo-ai-bulk-add-button.html [hunk 3]
```diff
@@ -53,6 +55,13 @@
...
+                type: Boolean,
+                value: function() {
+                    return Nuxeo.UI.config && Nuxeo.UI.config["nuxeo.ai.insight.client.enable"] || false;
```
Comment (nelsonsilva) [inline nuxeo-ai-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-ai/nuxeo-ai-bulk-add-button.html:63]: should be ` Nuxeo.UI.config.ai.insight.client.enable` no ? 🤔

File: nuxeo-ai-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-ai/nuxeo-ai-bulk-add-button.html (hunk 2 above)
Comment (andreacornaglia) [reply nuxeo-ai-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-ai/nuxeo-ai-bulk-add-button.html:60]: @tiagocardoso thanks! I'll go with Nelson suggestion of not using a property and directly using a function instead.

**Pull request merged**